### PR TITLE
Final Touches

### PR DESCRIPTION
**Pull request by for-ashraf**
At this point, I have a fully working app that uses styling to make it more user-friendly to make the layout work, according to the required wireframe. I achieved styling through CSS and didn't apply any framework.

### CODE REVIEW

**Review by KDlamini — COMMENTED** [review]
## Status: Invalid 🔴 

### Hi @for-ashraf 🙋‍♀️,  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project, that is not documented.  Please make sure that the README file looks professional and contains all that's needed for running the app with the API(s) it consumes. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
-# Leaderboard
-This is a Repository where I will manage leaderboard having name and score of a person.
+# To-Do List
+
+>In this project I have setup an application that will consume API to create a game and save and fetch players's scores and built with Webpack.
```
Comment (BenMukebo) [inline README.md]: - [ ] Let's try to be thorough in our documentation. This is the `Leaderboard` project, but your README contains information describing the `To-Do List`. I suggest you correct the title of your project. 👍🏻

File: README.md [hunk 2]
```diff
@@ -1,2 +1,53 @@
...
+👤 **Muhammad Ashraf**
+
+- GitHub: [@for-ashraf](https://github.com/for-ashraf)
+- LinkedIn: [Muhammad Ashraf](https://www.linkedin.com/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
```
Comment (BenMukebo) [inline README.md:40]: - [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @for-ashraf,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [ ]  The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/DynamicHTML.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 export const populateScores = (scoresArray) => {
   const scoresTable = document.getElementById('scores_table');
-  scoresTable.innerHTML = '<th><tr><td>Player</td><td>Score</td></tr></th>';
+  scoresTable.innerHTML = '<th><tr style="color:Orange;"><td>Player</td><td>Score</td></tr></th>';
   for (let i = 0; i < scoresArray.length; i += 1) {
     const tr = document.createElement('tr');
+    tr.classList.add('rowStyle');
     const td1 = document.createElement('td');
     const td2 = document.createElement('td');
     td1.append(`${scoresArray[i].user}`);
```
Comment (BenMukebo) [inline src/DynamicHTML.js:9]: - [ ] I'm facing this error message when I run your application on my computer, Kindly fix this 👇🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @for-ashraf👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**